core/src/main/resources/schema/infinispan-config-11.0.xsd

```diff
@@ -1401,17 +1403,19 @@
     <xs:attribute name="size" type="xs:long" default="-1">
       <xs:annotation>
         <xs:documentation>
-          Defines the size of the data container as a long. Eviction occurs
-          when the number of entries exceeds the size.
+          Deprecated, use the "max-size" attribute instead. Defines the size
```

Comment:
The whole ```"memory-object"``` type is already deprecated, we should not mention the replacament here because it lacks scope: ```use the "max-size" attribute instead``` appear to refer to the ```memory-object``` itself

Reply:
thanks @gustavonalle I've reverted those extra "deprecation" comments I made.